hnn_qt5.py

```diff
@@ -62,7 +62,11 @@ def parseargs ():
 debug = dconf['debug']
 testLFP = dconf['testlfp'] or dconf['testlaminarlfp']
 
-defncore = multiprocessing.cpu_count() # default number of cores
+# get default number of cores
+try:
+  defncore = len(os.sched_getaffinity(0))
```

Comment:
This will get the number of cores when limited by cgroups (e.g. SLURM on Oscar)

Reply:
OK, that sounds good. Assume outside of limited cgroups, works same as before.